plugins/addClassesToSVGElement.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+exports.type = 'full';
+
+exports.active = true;
```

Comment:
Can you give me a hint about how to use this plugin from the javascript API after I disable it by default? Specifically, I'm using it in conjunction with [svgo-loader](https://github.com/pozadi/svgo-loader) in webpack:

```JavaScript
{
    test: /icon-(.*)\.svg$/,
    loaders: [
        'ngtemplate?' + querystring.stringify({
            relativeTo: 'svg/',
        }),
        'raw',
        'svgo-loader?' + JSON.stringify({
            plugins: [
                {addClassesToSVGElement: {className: 'patreon-icon-svg'}}
            ]
        })
    ]
}
```

When I change `exports.active = true;` in `addClassesToSVGElement.js`, I can no longer use it from webpack — I assume there is some other parameter I need to pass in the configurations object to re-enable it, but I can't figure out what it is.